examples/linear_model/plot_logistic_l1_l2_sparsity.py

```diff
@@ -40,7 +40,7 @@
 
 # Set regularization parameter
 for i, (C, axes_row) in enumerate(zip((1, 0.1, 0.01), axes)):
-    # turn down tolerance for short training time
+    # turn up tolerance for short training time
```

Comment:
Nit: I find the phrase "turn up" to be a little informal. What do you think of using "Increase" here?

```suggestion
    # Increase tolerance for short training time
```

Reply:
Using "Increase" definitely works better here, @thomasjpfan !

And given this is an example (which might allow sparing a few extra words), I would go as far as replacing the entire line with "Increasing tolerance reduces the training time, at the expense of model accuracy".